src/ui/Toggle/ToggleContainer.tsx

```diff
@@ -7,7 +7,7 @@ import {
 } from './ToggleContext';
 
 export interface ToggleContainerProps extends ToggleContextInterface {
-  children?: React.ReactElement;
+  children?: React.ReactElement | null;
```

Comment:
children can be null?

Reply:
yeah, it should be nullable because the default value is null

Reply:
But, toggle container should have childeren, no?
```
<ToggleContainer>
   ... come UI component
</ToggleContainer>
```

`<ToggleContainer/>` is useless, no?